### PR TITLE
BUG: Mark translatable strings in Markups module

### PR DESCRIPTION
**Pull request by mhdiop**
Mark some remaining translatable strings in Markups module
* Slicer/SlicerLanguagePacks#12

### CODE REVIEW

File: Modules/Loadable/Markups/Widgets/qMRMLMarkupsCurveSettingsWidget.cxx [hunk 1]
```diff
@@ -45,8 +45,8 @@ class qMRMLMarkupsCurveSettingsWidgetPrivate
 public:
   qMRMLMarkupsCurveSettingsWidgetPrivate(qMRMLMarkupsCurveSettingsWidget &widget);
 
-  static const char* getCurveTypeAsHumanReadableString(int curveType);
-  static const char* getCostFunctionAsHumanReadableString(int costFunction);
+  static std::string getCurveTypeAsHumanReadableString(int curveType);
+  static std::string getCostFunctionAsHumanReadableString(int costFunction);
```
Comment (lassoan) [inline Modules/Loadable/Markups/Widgets/qMRMLMarkupsCurveSettingsWidget.cxx:49]: To follow Qt conventions
```suggestion
  static QString costFunctionAsDisplayableString(int costFunction);
```

**Comment by lassoan** [comment]
@mhdiop Could you please have a look at the comments and update the pull request accordingly? Thank you!

**Comment by mhdiop** [comment]
Thank you @lassoan for the reminder. I'll update and submit it in a few moments.

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h [hunk 2]
```diff
@@ -1029,6 +1032,14 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
 
   std::string PropertiesLabelText;
 
+  /// Store markup type GUI display name
+  /// It's displayed to the user and is therefore translated to the application language
```
Comment (jcfr) [inline Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h:1036]:
```suggestion
  /// \note It's displayed to the user and is therefore translated to the application language
```

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h [hunk 3]
```diff
@@ -1029,6 +1032,14 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
...
+  std::string TypeDisplayName;
+
+  /// Store markup short name
+  /// It's displayed to the user and is therefore translated to the application language
```
Comment (jcfr) [inline Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h:1040]:
```suggestion
  /// \note It's displayed to the user and is therefore translated to the application language
```

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsClosedCurveNode.cxx [hunk 4]
```diff
@@ -18,6 +18,7 @@
 #include "vtkMRMLMarkupsClosedCurveNode.h"
 
 // MRML includes
+#include "vtkMRMLI18N.h"
 #include "vtkCurveGenerator.h"
 #include "vtkCurveMeasurementsCalculator.h"
```
Comment (jcfr) [inline Modules/Loadable/Markups/MRML/vtkMRMLMarkupsClosedCurveNode.cxx:23]:
```suggestion
#include "vtkCurveGenerator.h"
#include "vtkCurveMeasurementsCalculator.h"
#include "vtkMRMLI18N.h"
```

**Review by lassoan — APPROVED** [review]
It all looks good, thank you!

**Pull request merged**